frontend/website/pages/docs/tokens.mdx

```diff
@@ -0,0 +1,112 @@
+import Page from '@reason/pages/Docs';
+export default Page({title: "Tokens"});
+
+# Tokens  
+
+Using the Coda CLI (command-line interface) is the primary way to create new token accounts, create new tokens, and mint tokens on the Coda blockchain. 
+
+In this section, you will find out how to use the Coda CLI to create new tokens, create new token accounts, and mint non-default tokens!
```

Comment:
This sentence is a little repetitive given the prior one. Can we combine these somehow?